{"name": "3. Выход из аккаунта из ЛК", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <tests.test_user_profile.TestUserProfile object at 0x000001DE7E189D00>\nauthorized_user = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"52db6bd6-fccb-4d42-bb12-43823720c47c\")>\nprofile_page = <pages.profile_page.ProfilePage object at 0x000001DE00074510>\n\n    @allure.title(\"3. Выход из аккаунта из ЛК\")\n    def test_user_logout(self, authorized_user, profile_page):\n        with allure.step(\"1. Открыть страницу профиля\"):\n            profile_page.open()\n>           WebDriverWait(authorized_user, 10).until(\n                EC.visibility_of_element_located(ProfilePageLocators.PROFILE_FORM)\n            )\n\ntests\\test_user_profile.py:53: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"52db6bd6-fccb-4d42-bb12-43823720c47c\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x000001DE00046980>, message = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\nvenv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:101: TimeoutException"}, "steps": [{"name": "1. Открыть страницу профиля", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\tests\\test_user_profile.py\", line 53, in test_user_logout\n    WebDriverWait(authorized_user, 10).until(\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^\n        EC.visibility_of_element_located(ProfilePageLocators.PROFILE_FORM)\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1748200165315, "stop": 1748200176879}], "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1748200165315, "stop": 1748200176881, "uuid": "ba9df811-4e48-4e93-9baa-412e6c48fbf4", "historyId": "0af322d8fb3d4e4893ca58535b295ecb", "testCaseId": "212a7779ac946c52ccc9547636e8ad80", "fullName": "tests.test_user_profile.TestUserProfile#test_user_logout", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('driver')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_profile"}, {"name": "subSuite", "value": "TestUserProfile"}, {"name": "host", "value": "Gemini_78"}, {"name": "thread", "value": "19464-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_profile"}]}